src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -206,7 +206,7 @@
     </form>
 
     <script type="text/template" id="template-pane">
-        <nav class="nav-apps plugins">
+        <nav class="nav-apps plugins nav-apps-narrow-manual">
```

Comment:
Consider dropping `-manual` from the class name since there is only one class now.